pkg/vcs/vcs.go

```diff
@@ -147,6 +147,7 @@ const (
 
 type BisectEnv struct {
 	Compiler     string
+	Ccache       string
```

Comment:
> Do we need it here? We don't change it based on kernel version and it does not even seem to be used.

you are right. It can be removed from here. Fixed now.

Reply:
This is not done either.

Reply:
Same here.